Tests/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+option(LOADER_TEST "Include the protocols library." OFF)
+option(HTTPSCLIENT_TEST "Include the protocols library." OFF)
+option(CRYPTOGRAPHY_TEST "Include the protocols library." OFF)
+
+if(BUILD_TESTS)
+    add_subdirectory(unit)
+endif()
+
+if(CRYPTOGRAPHY_TEST) 
```

Comment:
@bramoosterhuis these tests have moved to the ThunderClientLibraries as the cryptography tests moved there..Can you remove them here again

Reply:
Done!